src/datahike/pull_api.cljc

```diff
@@ -1,12 +1,13 @@
 (ns ^:no-doc datahike.pull-api
   (:require
-    [datahike.db :as db]
-    [datahike.datom :as dd]
-    [datahike.pull-parser :as dpp #?@(:cljs [:refer [PullSpec]])])
-    #?(:clj
-      (:import
-        [datahike.datom Datom]
-        [datahike.pull_parser PullSpec])))
+   [datahike.db :as db]
+   [datahike.datom :as dd]
+   #?@(:cljs [datalog.parse.type :refer [PullSpec]])
```

Comment:
Isn't this supposed to be `.parser.`?
And also `PullSpec` does not exist in `datalog.parser.type`.

Reply:
Also, is `type` the right name here?

Reply:
You're correct, it is the wrong namespace.